src/libxively/event_dispatcher/xi_event_handle_typedef.h

```diff
@@ -115,6 +115,11 @@ typedef struct xi_event_handle_s
     uint8_t target_tid;
 } xi_event_handle_t;
 
+#define xi_make_empty_event_handle( target_tid )                                         \
```

Comment:
don't remember about our conventions but isn't lowercase misleading to a function? Wouldn't it be better as
XI_EMPTY_EVENT_HANDLE( target_tid )
?